src/components/providers/UserProvider.tsx

```diff
@@ -13,7 +13,7 @@ interface Props {
 const UserProvider: FC<Props> = ({ children }: Props) => {
   const firebase = useFirebase();
   const [acceptsEmail, setAcceptsEmail] = useState<boolean | undefined>(
-    false,
```

Comment:
Is this change intentional?

Reply:
Yes, since it generated an error in the pagination, but I will set it back to false, and perform the infinity scroll.